resources/windows/wix/service.pxp-agent.wxs.erb

```diff
@@ -58,10 +58,9 @@
                         Value="1"
                         Type="integer" />
           <RegistryKey Key="AppExit">
-            <!-- Stop the service completely on exit(2) (Invalid configuration) otherwise restart with NSSM service throttling -->
+            <!-- Stop the service completely for all exit codes -->
             <!-- nssm AppExit codes reference https://nssm.cc/usage#exit -->
-            <RegistryValue Value="Restart" Type="string" />
-            <RegistryValue Name="2" Value="Exit" Type="string" />
+            <RegistryValue Value="Exit" Type="string" />
```

Comment:
This requested behavior doesn't match what the commit comment says this is trying to address?

Reply:
What part of the commit message contains the mismatch you're referring to?

Reply:
The message states a specific scenario, and the code is not scenario specific.

Reply:
Oh I see the confusion. I'll reword the message.

Reply:
I reworded the message.

Reply:
LGTM. 👍

Reply:
I'm tempted to say we should restart on exit code 1. But I don't have specific scenarios where that should be necessary, that'd purely be crash recovery.

Reply:
Do you want me to go ahead and add a restart line for exit code 1?

Reply:
Yeah, let's try that. The others are intentional exits because something isn't setup right. 1 probably means a crash or unexpected failure.

Reply:
Updated the PR.